examples/cli_example_tiny_ophys_training.py

```diff
@@ -61,41 +53,28 @@
 generator_param["pre_post_omission"] = 0
 
 # Those are parameters used for the network topology
-network_param["type"] = "network"
 network_param["name"] = "unet_single_1024"
 # Name of network topology in the collection
 
 # Those are parameters used for the training process
-training_param["type"] = "trainer"
 training_param["name"] = "core_trainer"
 training_param["learning_rate"] = 0.0001
 training_param["loss"] = "mean_squared_error"
-training_param["output_dir"] = "/Users/jeromel/Desktop/tmp"
+training_param["output_dir"] = str(pathlib.Path(__file__).parent.absolute())
```

Comment:
Same comment about using `argparse` to let the user specify the output.

I suppose an alternative would be to add a print statement informing the user that the script was about to write output to this directory.

Reply:
I am going to pass on this as I would prefer folks to use the CLI than this.